parsel/selector.py

```diff
@@ -290,7 +325,18 @@ def re(self, regex, replace_entities=True):
         """
         return extract_regex(regex, self.get(), replace_entities=replace_entities)
 
-    def re_first(self, regex, default=None, replace_entities=True):
+    @typing.overload
+    def re_first(self, regex: Union[str, Pattern[str]], default: None = None,
+                 replace_entities: bool = True) -> Optional[str]:
+        pass
```

Comment:
I see Codecov is complaining about these. See https://github.com/nedbat/coveragepy/issues/970#issuecomment-612602180

Reply:
Good point. Done.